lola.html

```diff
@@ -0,0 +1,875 @@
+      </p>
+
+      <p>Open Issue: The actor URL with a URL parameter approach offered above seems like an OK approach but there are others possible.</p>
+
+      <ul>
+        <li>
+          Making the source responsible: The destination could try to provide the source server with a list of object-level redirect mappings. The source server would try to keep this mapping list around for a year to do specific redirects.
+        </li>
+        <li>
+          Making 3rd parties responsible: An agent not involved as either the source or the destination may follow a link to a object that has moved.  The agent making the request for a specific moved object could receive just the redirect to the local Actor, showing the movedTo value.  The agent could then take the responsibility for trying to find the moved object by fetching likely objects hoping to find an object ID match in breadcrumbs.
+        </li>
+
+        <li>
+          Making the destination responsible in a different way: instead of the URL parameter suggested above, there could be a “find object URL that the destination hosts. 
+        </li>
+
+      </ul>
+
+      <h3>Third-Party Behavior</h3>
+
+      <p>
+        A server not involved in an account migration can be aware of this specification and might find the following considerations useful.
+      </p>
+
+      <h4>Following up a ‘Like’ of moved object</h4>
+
+      <p>
+        When a third-party server has a ‘Like’ activity pointing to a moved activity, it need not take any action to fix up ‘Like’ links.  However sometimes a user wants to dereference a Like and find the original article.  When the user is browsing their Likes via Web GUI, the server can attempt to use the original location and object ID of the Liked activity to find the activity’s new location.  
+      </p>
+
+      <p>
+        Open Issue: will the server even detect that the Like reference has moved?  It may only present a link to the remote object and not ever try to dereference it.
+      </p>
+
+      <p>
+        When a user is browsing Likes via ActivityPub client or using another C2S protocol, a smart client may find itself in the same situation - the user just clicked on a Like and the thing liked has moved.
+      </p>
+
+      <p>
+        The agent requesting the Liked item may receive a generic redirect to an Actor, not an Object (this won’t be obvious until the referenced resource is fetched).  If the agent finds an Actor, it may fetch a number of objects hoping to find one with a breadcrumb matching the original Like reference.
+      </p>
+
+      <p>
+        Open Issue: should the Like activity then be rewritten with the new location? Can a client do this - is there an affordance to update a Like? 
+      </p>
+
+    </section>
+  </body>
```

Comment:
We need a conformance section. I think this is all that's necessary:

```
<section id="conformance">
      <p>
        This is required for specifications that contain normative material.
      </p>
    </section>
```

ReSpec will actually fill it in with information about conformance, including the RFC 2119 conformance.